generators/profilegen/src/resources/templates/individual/client_template.template

```diff
@@ -21,15 +21,5 @@ module Azure::<%= @class_name %>::Profiles::<%= @profile_name %>::Mgmt
       super(options)
     end
 
-    def credentials
-      if @credentials.nil?
```

Comment:
shouldn't this change have come as part of the merge? with merged PR https://github.com/Azure/azure-sdk-for-ruby/pull/1090/files ?
I'm curious on what else we could have missed in the merge, therefore the question.